File: SourceCode/Projects/open.fda.gov/ApiDalc/DataObjects/FeatureObject.vb
Imports Newtonsoft.Json

<Serializable>
Public Class FeatureObject

    Public Property Type As String
    Public Property Properties As FeaturePropertyObject
    Public Property Teometery As GeometryObject

End Class

<Serializable>
Public Class FeaturePropertyObject

    Public Property GEO_ID As String
    Public Property STATE As String
    Public Property NAME As String
    Public Property LSAD As String
    Public Property CENSUSAREA As Double


End Class

<Serializable>
Public Class GeometryObject
    Public Property type As String
    Public Property coordinates As List(Of LatLon)
End Class

<Serializable>
Public Class LatLon
    Public Property loc As List(Of Double)
End Class

File: SourceCode/AspCoreVersion/src/ShopAware.Core/DataObjects/FeatureObject.cs
using System.Collections.Generic;

namespace ShopAware.Core
{
    public class FeatureObject
    {
        #region Properties

        public string Type { get; set; }

        public FeaturePropertyObject Properties { get; set; }

        public GeometryObject Teometery { get; set; }

        #endregion
    }

    public class FeaturePropertyObject
    {
        #region Properties

        public string GEO_ID { get; set; }

        public string STATE { get; set; }

        public string NAME { get; set; }

        public string LSAD { get; set; }

        public double CENSUSAREA { get; set; }

        #endregion
    }

    public class GeometryObject
    {
        #region Properties

        public string type { get; set; }

        public List<LatLon> coordinates { get; set; }

        #endregion
    }

    public class LatLon
    {
        #region Properties

        public List<double> loc { get; set; }

        #endregion
    }
}